src/tools/sdk/_windows.md

```diff
@@ -0,0 +1,33 @@
+#### Install using a setup wizard
```

Comment:
The style of these 4th-level headers looks strange to me under the smaller font of the tabs. Let's see if it looks better once we add introductory text.